client/my-sites/plans/index.js

```diff
@@ -47,23 +47,8 @@ export default function() {
 		clientRender
 	);
 	page( '/plans/features/:feature/:domain', plansController.features, makeLayout, clientRender );
-	page(
-		'/plans/my-plan',
-		siteSelection,
-		sites,
-		navigation,
-		currentPlanController.currentPlan,
-		makeLayout,
-		clientRender
-	);
-	page(
-		'/plans/my-plan/:site',
-		siteSelection,
-		navigation,
-		currentPlanController.currentPlan,
-		makeLayout,
-		clientRender
-	);
```

Comment:
This is prettier inlining the function invocation now that the arguments are shorter.